src/components/__tests__/AffectExpandableForm.spec.ts

```diff
@@ -152,7 +152,24 @@ describe('AffectExpandableForm', () => {
     expect(affectednessOptions).toStrictEqual(affectAffectedness);
     await affectednessSelectEl.find('select').setValue('AFFECTED');
     const resolutionOptions = resolutionSelectEL.props('options');
-    expect(resolutionOptions).toStrictEqual(AffectednessResolutionPairs['AFFECTED']);
+    expect(resolutionOptions).toStrictEqual(possibleAffectResolutions('IMPORTANT')['AFFECTED']);
+  });
+
+  it('should allow DEFER with impact LOW', async () => {
```

Comment:
Maybe we should add another test `should not allow DEFER without impact LOW` to make sure the restriction is correct